playbooks/ansible/roles/sit.glusterfs/tasks/new_volume.yml

```diff
@@ -9,16 +9,16 @@
   run_once: yes
   when: volume.type == 'replicate'
```

Comment:
If only one type of volume is created, this should be removed and we don't need to pass the `volume.type` argument.

Reply:
Sure.